packages/sdk-core/src/ConstantFlowAgreementV1.ts

```diff
@@ -132,6 +139,71 @@ export default class ConstantFlowAgreementV1 {
         }
     };
 
+    /**
+     * @dev Get flow operator data.
+     * @param superToken the superToken of the agreement
+     * @param sender the sender
+     * @param flowOperator the flowOperator
+     * @param providerOrSigner a provider or signer object
+     * @returns {Promise<IWeb3FlowOperatorData>} Web3 Flow info object
+     */
+    getFlowOperatorData = async ({
+        superToken,
+        sender,
+        flowOperator,
+        providerOrSigner,
+    }: IGetFlowOperatorDataParams): Promise<IWeb3FlowOperatorData> => {
```

Comment:
FYI, I never use the "I" prefix. Do we have this specified in a code guidelines somewhere?

Reply:
no, but I think it would be good to have a level of consistency across the entire codebase